src/Platform.ts

```diff
@@ -58,7 +59,10 @@ export function program<Model, Msg>(
     distinctUntilChanged()
   )
 
-  const sub$ = model$.pipe(switchMap(model => subscriptions(model)))
+  const sub$ = model$.pipe(
+    startWith(init[0]), // This is needed to avoid swallowing of initial model in some situations
```

Comment:
`startWith` is indeed necessary for subs ([it's been fixed in previous version](https://github.com/gcanti/elm-ts/pull/11)), but added tests are not reflecting this need, i.e. they pass with or without `startWith`.
I can try to come up with test scenario that reflects this need if it is ok?

Reply:
@minedeljkovic I'm going to close this PR because the problem is un-reproducible with tests (I tried a lot of times without success...)

Furthermore, it seems that the issue I had in my application (which led me to add `startWith` operators to streams) vanished when I upgraded some other project's dependencies...

Reply:
Ok.
Anyhow, I'll try to come up with a test that reproduces the problem that existed with previous version

Reply:
Ok, then I won't close the PR and I'll add a "stale" label so you can try it

Reply:
I cannot reproduce #10 with new version. I have no other explanation than something has changed in `rxjs`.
I believe this PR can be safely abandoned.

Reply:
I came to the same conclusion.

Thank you Milos